internal/datastore/invite_request.go

```diff
@@ -466,3 +466,37 @@ func (d *delphisDB) AccessRequestIterCollect(ctx context.Context, iter Discussio
 
 	return requests, nil
 }
+
+func (d *delphisDB) GetInvitedTwitterHandlesByDiscussionIDAndInviterID(ctx context.Context, discussionID string, invitingParticipantID string) ([]*string, error) {
```

Comment:
It makes sense for me to return a list of string instead of a more complex object, mainly for two reasons:
- It optimizes the query as we only need one single field. Also the design is open for future refactorings to fetch more data if needed.
- The proper object to be returned would be a DiscussionInvite. However, such type doesn't contain any info about the Twitter nickname related to the invited user, and I didn't want to refactor that as it's part of other code and probably would not even be a good design choice